.yarnrc.yml

```diff
@@ -6,6 +6,8 @@ logFilters:
 
 nodeLinker: node-modules
 
+npmRegistryServer: "https://registry.npmjs.org"
```

Comment:
Change to `http://localhost:4873/` whenever you want to use verdaccio as a registry for yarn (and lerna). Assuming you have verdaccio running obv.